<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeidenhershSimulateDialog</class>
 <widget class="QDialog" name="HeidenhershSimulateDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tool</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dspnAngle">
          <property name="suffix">
           <string>Â°</string>
          </property>
          <property name="minimum">
           <double>5.000000000000000</double>
          </property>
          <property name="maximum">
           <double>45.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>17.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Tip grind</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dspnTipGrind">
          <property name="suffix">
           <string>mm</string>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>0.150000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="SimulateHeidenhershGraphicsView" name="gvDisplay"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SimulateHeidenhershGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>simulateheidenhershgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
